modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -447,6 +447,18 @@ object ElasticQuery {
   final def matchPhrase(field: String, value: String): MatchPhraseQuery[Any] =
     MatchPhrase(field = field, value = value, boost = None)
 
+  /**
+   * Constructs an instance of [[zio.elasticsearch.query.MultiMatchQuery]] using the specified parameter.
+   * [[zio.elasticsearch.query.MultiMatchQuery]] query builds on the `match` query to allow multi-field queries.
+   *
+   * @param value
+   *   the text value to be matched
+   * @return
+   *   an instance of [[zio.elasticsearch.query.MultiMatchQuery]] that represents the multi match query to be performed.
+   */
+  final def multiMatch(value: String): MultiMatchQuery[Any] =
+    MultiMatch(fields = None, value = value)
+
```

Comment:
Because fields are optional and type-safe method is now in `HasFields` trait, i suppose we don't need `multiMatch` type-safe method?

Reply:
If you change `fields` inside of query, this will then be: `fields=Chunk.empty`